registry/quilt_server/views.py

```diff
@@ -1353,7 +1367,7 @@ def access_delete(owner, package_name, user):
             "Cannot revoke the owner's access"
         )
 
-    if HAVE_PAYMENTS and user == PUBLIC:
+    if HAVE_PAYMENTS and not TEAM_ID and user == PUBLIC:
```

Comment:
my main feedback for this block is readability. I realize there's no `else` here, but I think `if user == PUBLIC` would be cleaner on a separate line and address why no else in a comment. Everywhere else in this PR (and now in the frontend, too), the plans logic is much clearer. I would also like to see separate groups of `PaymentPlan`s as I think it's clear that we have two disjoint sets and they belong in separate dicts. Not religious about this but if anyone goes back to touch this code they should see to distinct sets of plans.

Reply:
Sure